userguide/snippets/es60.rst

```diff
@@ -126,8 +131,8 @@ secures the rail in place. Repeat this process on the other side.
 Mount Rack Rails
 ^^^^^^^^^^^^^^^^
 
-Install four cage nuts in the rack, one for each bracket. The lock
-screw for the bracket is already installed in the cage nut. Place the
+Install the four cage nuts with locking screws in the rack. Place one
```

Comment:
"the" is not needed here, and might even confuse the reader if they get more than four nuts in a package.  The phrasing on the last part of the sentence is confusing also, implying there is a "locking screw", rather than a screw that holds the unit in place.